dashboard/model_layer.py

```diff
@@ -164,9 +164,11 @@ def enrich_session_info(se):
     m, s = divmod(duration_seconds, 60)
     h, m = divmod(m, 60)
     se.duration = "%d:%02d:%02d" % (h, m, s)
-    se.client_bytes_sent = se.client_bytes_sent / 1024
-    se.client_bytes_received = se.client_bytes_received / 1024
-    se.data_transferred = se.client_bytes_sent + se.client_bytes_received
+    se.data_sent = humanize.naturalsize(se.client_bytes_sent)
+    se.data_received = humanize.naturalsize(se.client_bytes_received)
+    se.data_transferred = humanize.naturalsize(
+        se.client_bytes_sent + se.client_bytes_received
+    )
```

Comment:
`humanize.naturalsize(1024 * 1024 * 1024)` returns `'1.1 GB'`, which is incorrect - it should be 1 GB.
`binary=True` option solves this.

Extracting simple utility function for humanization and adding some tests would be cool - than it would be clear what format these strings are returned in, without having to look in library docs (which are a bit messy). Also, we would be sure it works the way we want :)

Reply:
with kB MB and GB its confusing, there are 3 representation standards: Si, Iec, Jedec.
According to Si this calculates it right.
If we want to be precise and not confuse anyone, we in this case should use notation `GiB` according to Iec.

Reply:
actually same thing is happening on Mysterion